code/src/Icons.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import "./icons.css";
+import data from "./data.json";
+
+export const Icons = props => (
+  <div>
+    <div className="icons">
+      <div className="iconsContainer">
+        <img
+          className="iconHeart"
+          src="./icons/heart.svg"
+          alt="Heart icon for favourite"
+        ></img>
```

Comment:
Image tags are self-closing and also, when using images in the public folder, use paths starting with `/`.. so this should be:

```
<img
  className="iconHeart"
  src="/icons/heart.svg"
  alt="Heart icon for favourite" />
```